src/assets/styles/global.css

```diff
@@ -1,3 +1,6 @@
 * {
   box-sizing: border-box;
 }
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
seems like unnecessary style files. did you use Tailwindcss? @passion0401lad

Reply:
Yeah, as you know some HTML tags have default margin or padding values originally and it is so inconvenient for me to correct every time by the way I wanna set the default value to zero. So I did like above.